### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ivanovUA**
Added dockerfile

### CODE REVIEW

**Comment by mprochowski** [comment]
Maybe it's better to use `FROM scratch` instead of alpine image.

**Comment by neclimdul** [comment]
While a scratch image is great, a version with a shell would be useful for GitLab's runner which blows up without a shell.

Example of Google Kaniko adding an image with a shell to solve this.
https://gitlab.com/gitlab-org/gitlab-foss/-/issues/45512

**Comment by mprochowski** [comment]
In my opinion, docker image should be as small as possible. If someone wants to use it in another project (like GitLab's runner), they can always create their own multi-step image with app from official image. 

Let's create simple things, not complex just because someone maybe need it in the future :smiley:

**Comment by neclimdul** [comment]
So yeah, its possible to make it that small. It doesn't match my needs as well as some of the people discussing in #8 who are already trying to use this in gitlab pipelines. From my needs it feels be like only making a arm build because that's the smallest possible.

If having the main image created `from scratch` is a hard requirement for getting this accepted that's fine. I was offering Google's approach as a solution to make this as useful.  TL;DNR, they have a scratch image that works for their pipelines but also a `:dev` tag with busybox so you can create a container with a shell for debugging but it also supports gitlab pipelines.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.13.8
```
Comment (ntavelis) [inline Dockerfile:1]: On the release file, version 1.15 is used. Use also 1.15 here ?

File: README.md [hunk 2]
```diff
@@ -28,5 +28,13 @@ downloads it from Github if it changed since the last run. If you want to avoid
 the HTTP round-trip, use `--local`. To force a database update without checking
 for a project, use `--update-cache`.
 
+Building Docker image
+
+    $ docker build -t local-php-security-checker
```
Comment (antavelis) [inline README.md:33]:
```suggestion
    $ docker build -t local-php-security-checker .
```

Comment (antavelis) [reply README.md:33]: To build from project's root folder, docker build command requires the path to the dockerfile

File: Dockerfile (hunk 1 above)
Comment (ivanovUA) [reply Dockerfile]: Updated to 1.15

**Comment by ivanovUA** [comment]
Updated build image to Go v15 and base image - scratch with certificates.

**Review by wow-apps — APPROVED** [review]

**Comment by ignatenkovnikita** [comment]
HI! Why MR not merge ?

**Comment by AlexSkrypnyk** [comment]
Tested this - works great!

Got a project using Drupal 7.40 to test:


___www_local-php-security-checker-docker__o_o_

**Comment by fabpot** [comment]
Closing as I'm going to archive this repository. Use `composer audit` instead.

**Pull request closed without merging**